src/hooks/useSnackbar/useSnackbar.tsx

```diff
@@ -42,9 +45,11 @@ const SNACKBARS_LIMIT = 3
 export const SnackbarProvider: React.FC = ({ children }) => {
   const [snackbars, setSnackbars] = useState<SnackbarsState[]>([])
 
-  const displaySnackbar = useCallback(({ timeout, ...args }: DisplaySnackbarArgs) => {
-    const id = createId()
-    setSnackbars((s) => [...s, { id, ...args }])
+  const displaySnackbar = useCallback(({ customId, timeout, ...args }: DisplaySnackbarArgs) => {
+    const id = customId || createId()
```

Comment:
Do we actually need this `customId`? We never using it

Reply:
True, it's a left over from the previous implementation, will remove it